docs/examples/tutorials-and-example-projects.md

```diff
@@ -45,6 +45,11 @@ possible and jump-start your own projects.
       <td><a href="https://docs.semaphoreci.com/examples/java-spring-continuous-integration/">Spring</a></td>
       <td><a href="https://github.com/semaphoreci-demos/semaphore-demo-java-spring">semaphore-demo-java-spring</a></td>
     </tr>
+    <tr>
+      <td>Java</td>
```

Comment:
Change this to "Android, Java" so it's easier to notice that this is an Android demo